torch/ao/quantization/pt2e/utils.py

```diff
@@ -86,7 +86,8 @@ def _find_q_dq_node_for_user(
     q_node = None
     if (
         dq_node.args[0].op == "call_function"  # type: ignore[union-attr]
-        and dq_node.args[0].target in _QUANTIZE_OPS  # type: ignore[union-attr]
+        and isinstance(target := dq_node.args[0].target, torch._ops.OpOverload)  # type: ignore[union-attr]
```

Comment:
In this one case, you could get rid of the `# type: ignore[union-attr]` with an additional check in the if statement for the type of `dq_node.args[0]`.